### ISSUE TITLE
Error-message and Line-Number is NOT related to document 

### ISSUE THREAD

**Issue by aotto1968** [doxygen/doxygen#9409]
**Describe the bug**
I get an error-message

```
…/gen/libmkkernel.doc:1330: warning: Invalid list item found
…/gen/libmkkernel.doc:5667: warning: Invalid list item found
…/gen/libmkkernel.doc:5777: warning: Invalid list item found
…/gen/libmkkernel.doc:5922: warning: Invalid list item found
…/gen/libmkkernel.doc:5930: warning: Invalid list item found
```
but the document mentioned `…/gen/libmkkernel.doc` has **only** 1000 lines 

```
> wc …/gen/libmkkernel.doc
 1000  2715 46793 …/gen/libmkkernel.doc
```

The problem seems to be that a lot of `\copydoc` documentation is included in the _final_ document and the
line-number seems to be an _internal_ line-number and NOT the real one.

**Expected behavior**
I expect:

1. the right _line-number_
2. if the  right _line-number_ is **not*** possible a good solution would be to get some _context-lines_ to search for.

**To Reproduce**
This is part of a *large* project and a lot of documentation is included from different location, not easy possible to reproduce out of the project.

- the hint with the _context-line_ of the error would at least solve the immediate pain.

**Version**
1.9.3

**Comment by albert-github** [comment]
Line numbers are always a hard point especially when other parts are included. 
Based on the description / hints in the issue it looks like I was able to create a small example that shows the problem.

Example: [example.tar.gz](https://github.com/doxygen/doxygen/files/8935264/example.tar.gz)

**Comment by doxygen** [comment]
@aotto1968 Please verify if the problem is solved with the referenced commit.

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.9.5.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
